.azure-pipelines/release.yml

```diff
@@ -0,0 +1,93 @@
+# NOTE: this pipeline definition is not currently used to build releases of VFS for Git.
+# This is still done in the GVFS-Release-RealSign "classic" pipeline.

```

Comment:
I appreciate the comment. We could easily drop this from the PR but it can be nice to have as a reference for a future conversion.